src/lib/arrays.js

```diff
@@ -0,0 +1,104 @@
+const getNthElement = (index, array) => {
+  return array[index % array.length];
+};
+
+const arrayToCSVString = array => {
+  return array.join();
+};
+
+const csvStringToArray = string => {
+  return string.split(',', 5);
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  const newArray = array.concat(element);
+  return newArray;
+};
+
+const removeNthElement = (index, array) => {
+  return array.splice(index, 1);
+};
+
+const numbersToStrings = numbers => {
+  const numtoString = numbers.map(num => {
+    return String(num);
+  });
+  return numtoString;
+};
+
+const uppercaseWordsInArray = strings => {
+  function capitalizeWords(strings) {
+    return strings.map(element => {
+      return element.toUpperCase();
+    });
+  }
+  return capitalizeWords(strings);
+};
+
+const reverseWordsInArray = strings => {
+  return strings.map(function(A) {
+    return A.split('')
+      .reverse()
+      .join('');
+  });
```

Comment:
Convention with variable names is to make them something meaningful. In this case, `A` is very abstract and should be replaced with something like the singular name for the array being mapped over:
```
  return strings.map(string => {
    return string
      .split('')
      .reverse()
      .join('');
  });
```